engine/table/src/main/java/io/deephaven/engine/util/LayoutHintBuilder.java

```diff
@@ -105,9 +108,63 @@ static AutoFilterData fromString(String string) {
         }
     }
 
+    private static class ColumnGroup {
+
+        String name;
+        final List<String> children;
+        Color color;
+
+        ColumnGroup(String name, List<String> children, Color color) {
```

Comment:
constructor/methods should almost never be package-protected like this, but explicitly public/private/etc